target/ppc/translate/storage-ctrl-impl.c.inc

```diff
@@ -49,6 +49,20 @@ static bool trans_SLBIEG(DisasContext *ctx, arg_SLBIEG *a)
     return true;
 }
 
+static bool trans_SLBIA(DisasContext *ctx, arg_SLBIA *a)
+{
+    REQUIRE_64BIT(ctx);
+    REQUIRE_INSNS_FLAGS(ctx, SLBI);
+    REQUIRE_SV(ctx);
+
+#if !defined(CONFIG_USER_ONLY) && defined(TARGET_PPC64)
+    gen_helper_slbia(cpu_env, tcg_constant_i32(a->ih));
```

Comment:
ditto

Reply:
Fixed in 5e50d8f94c42642607568c8bd366e82739a7bab3